docs/docs/get-started/quickstart.md

```diff
@@ -181,6 +186,40 @@ nebari init azure --project projectname \
 
 <!-- TODO: Add commands for HPC and local cluster. -->
 
+## Validate (optional)
+
+After creating the `nebari-config.yaml` file, you can customize it. The Nebari package uses Pydantic for schema validation.  To ensure your customizations are valid, run:
+
+```bash
+nebari validate -c nebari-config.yaml
+```
+
+:::note
+Extensions built using the [Nebari Extension System][nebari-extension-system] may extend the Nebari schema.  If you are intending to use an extension and `nebari validate` returns an error `Extra inputs are not permitted`, ensure that the the correct versions of the extensions you intend to use are installed in your active Python environment.
+:::
+
+:::note
+This command is automatically run when you `deploy`.
+:::
+
+## Render (optional)
+
+You can generate the (Terraform) deployment workflow scripts with:
```

Comment:
I would very briefly expand this a bit, as this is the actual step that [renders/loads the stage classes/models](https://github.com/nebari-dev/nebari/blob/develop/src/_nebari/render.py#L29-L33) into physical files, and then writes the data accordingly, based on what has been updated making a checksum comparison between whats present in the user workspace x a temp render location.

Reply:
Done